Src/BootCamp.Chapter/Computer/DesktopComputer.cs

```diff
@@ -1,41 +1,22 @@
 ﻿namespace BootCamp.Chapter.Computer
 {
-    public class DesktopComputer
+    public abstract class DesktopComputer
     {
-        private Body _body;
-        public Body GetBody()
+        protected DesktopComputer()
         {
-            return _body;
-        }
-        
-        private Ram _ram;
-        public Ram GetRam()
-        {
-            return _ram;
-        }
-
-        private Cpu _cpu;
-        public Cpu GetCpu()
-        {
-            return _cpu;
-
-        }
-        private Gpu _gpu;
-        public Gpu GetGpu()
-        {
-            return _gpu;
+            AssemblePC();
```

Comment:
Desktop computer does not assemble itself. A factory does it. Please make it more logical.